spec/mail/network/delivery_methods/stdout_spec.rb

```diff
@@ -0,0 +1,63 @@
+# encoding: utf-8
+require 'spec_helper'
+
+describe "Stdout Delivery Method" do
+
+  before(:each) do
+    # Reset all defaults back to original state
```

Comment:
looks more like 'change global state so that this test passes', whatever test unsets them should clean them up